### DISCUSSION TITLE
Realesrgan GUI For Windows OS

### DISCUSSION THREAD

**Discussion by unamed666** [xinntao/Real-ESRGAN#852] [Show and tell]
## [DOWNLOAD HERE](https://github.com/eroge69/RealesrganGUI)

![image](https://github.com/user-attachments/assets/8832de5f-b88d-49a7-9acc-87e600326af4)
![image](https://github.com/user-attachments/assets/9346918c-a99e-4e5f-a7c9-7803331f2ca8)

**Comment by FlM122** [comment]
which model is the best for anime videos??? to ensure that the quality is as good as possible and that the speed is high . I found AnimeJaNai, but it makes the contour lines too sharp.

**Reply by unamed666** [reply]
Idk. I rarely use it for video and only tested it in realesergan
